notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        num_pixels = image_shape[0] * image_shape[1]
</pre><p>Add a comment above explaining how this computes the num of pixels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added a comment on it. Thanks